### ISSUE TITLE
Separating discussion from enhancement/bug tracking.

### ISSUE THREAD

**Issue by ClassicThunder** [craftworkgames/MonoGame.Extended#45]
Hey I find holding proper conversations here on GitHub to be a of a pain and it pollutes the issues page. I think we should set up a forum (I recommend https://disqus.com/) for discussions.

For example we can figure out the best way to handle the text field API on disqus and the enhancement issue could document the agreed up specs for the implementation.

**Comment by craftworkgames** [comment]
It has crossed my mind to have a discussion forum a couple of times, but I'm not sure if this is the best reason to do it.

Pros and cons of having a discussion forum:

### Pros
 - Perceived benefit of not polluting the issues page. To be honest I think I'd spend most of my time telling people not to discuss things here and people would discuss things here anyway.
 -  Conversations here on GitHub won't be a of a pain. Although, I'm not exactly sure what that means.
 - A discussion forum could be a place to discuss other things like games made with the library or other things?
 - ???

### Cons
 - Hosting costs of a disqus server? (Wait, are you sure you don't you mean [discourse](https://www.discourse.org/)). I normally use Azure but [digital ocean](https://www.digitalocean.com/community/tutorials/how-to-install-discourse-on-ubuntu-14-04) might be cheaper at $10 per month.
 - Issue discussion is fragmented. We'd then have 3 places to discuss things (github, [Trello](https://trello.com/b/Xi6Rfqhb/monogame-extended) and the new forums).
 - What domain would we host the forums? dylanwilson.net, craftworkgames.com or a new one monogameextended.net?
 - We'd constantly have to cross link everything. The github issue would link to Trello and forums, forums would link to github and Trello, Trello would link to github and forums. It makes my head spin.
 - ???

That said, I'm not against the idea of having discussion forums. I think there's a few clear benefits as the project gets bigger.

Maybe there's an alternative solution. We could ask the MonoGame team if they will setup a MonoGame.Extended category on [their forums](http://community.monogame.net/). The benefit being increased visibility of the project among MonoGame users.

In my opinion, github issues are the perfect place to discuss development (bugs and enhancements). They are discussions that support markdown and code snippets by design. They have the ability to link to the PR and other issues by typing numbers #44. They have labels (bug, enhancement, duplicate) and milestones right here on the page. If this isn't what github issue discussion was intended for I must be missing something.

**Comment by WardBenjamin** [comment]
Personally, I'm fine with having discussion in the github issues. Since I'm watching the repo, I get a comprehensive list of different discussions in almost real-time, which is especially nice when I'm waiting for a response on something. Hosting cost is not something that you should really have to incur for a project that is at this stage... if anything we could host something on github pages. I also really like the ability to easily link to pull requests and other issues.

TL;DR: I'm for keeping it here, but I'd like some more input(s).

**Comment by craftworkgames** [comment]
> if anything we could host something on github pages.

I love github pages but I don't think you could host a discussion forum on them? Do you know of a way that could work?

**Comment by craftworkgames** [comment]
I've been thinking about introducing a proper discussion forum lately.

[Discourse](https://www.discourse.org/) is probably the nicest modern forum software but the hosting costs are pretty hefty.

Any other ideas?

**Comment by craftworkgames** [comment]
I'd really like to get a discussion forum started this year. Hosting costs of discourse are still an issue unless I can raise some funds some how ([Patreon](https://www.patreon.com/) maybe?)

@tomspilman Another idea I'd like to toss around is getting a sub-forum for MonoGame.Extended within the [MonoGame Community](http://community.monogame.net/) perhaps?

**Comment by tomspilman** [comment]
@craftworkgames - Done...

http://community.monogame.net/category/extended

... let me know if you need any adjustments to that.

**Comment by craftworkgames** [comment]
Thanks @tomspilman. I really appreciate it :smile:
